RFC/src/RFC-0311_AssetTemplates.md

```diff
@@ -22,148 +22,148 @@ following conditions are met:
-| template_id                        | `u64`         | The template descriptor. See [Template ID](#template-id)                                                                               |
-| asset_expiry                       | `u64`         | A timestamp or block height after which the asset will automatically expire. Zero for arbitrarily long-lived assets                    |
-| **Validation Committee selection** |               |                                                                                                                                        |
-| committee_mode                     | `u8`          | The validation committee nomination mode, either `CREATOR_NOMINATION` (0) or `PUBLIC_NOMINATION` (1)                                                 |
-| committee_parameters               | Object        | See [Committee Parameters](#committee-parameters).                                                                                     |
-| asset_creation_fee                 | `u64`         | The fee the issuer is paying, in microTari, for the asset creation process                                                             |
-| commitment                         | `u256`        | A time-locked commitment for the asset creation fee                                                                                    |
-| initial_state_hash                 | `u256`        | The hash of the canonical serialisation of the initial template state (of the template-specific data)                                  |
-| initial_state_length               | `u64`         | Size in bytes of initial state                                                                                                         |
-| **template-specific data**         | Object        | Template-specific metadata can be defined in this section                                                                              |
-| **Signatures**                     |               |                                                                                                                                        |
-| metadata_hash                      | `u256`        | A hash of the previous three sections' data, in canonical format (`m`)                                                                 |
-| creator_sig                        | `u256`        | A digital signature of the message `H(R ‖ P ‖ RAID_ID ‖ m)` using the asset creator’s private key corresponding to the `issuer` PKH |
-| commitment_sig                     | `u256`        | A signature proving the issuer is able to spend the commitment to the asset fee                                                        |
-
-#### Committee parameters
-
-If `committee_mode` is `CREATOR_NOMINATION` the `committee_parameters` object is
+| Name                               | Type          | Description                                                  |
```

Comment:
All the formatting for this table is broken. The Colons should be nicely aligned as they were before.

Reply:
Hi @anselld, just copy the table only, paste it in [this tool](http://markdowntable.com/), format and copy the result. It nicely formats Markdown tables for easier reading when viewed with a pure text editor (although nothing is technically wrong with the table).

I see the table here
_If `committee_mode` is `PUBLIC_NOMINATION`, the `committee_parameters` object is:_
also has some misaligned colons at the end. You could also fix that.

Thanks!

Reply:
Hi @hansieodendaal , I don't see the problem... please explain.
Thanks and regards
Denise

Reply:
Hi @CjS77 , tables have been fixed - thanks for spotting that.